R/pkgs/covidcommon/R/config.R

```diff
@@ -0,0 +1,68 @@
+##'
+##'Map of configuration values, constructed via a call to `load_config`
+##'
+config <- NA
+
+##'
+##'Returns a map of configuration loaded from the config YAML
+##'@param fname Load configuration from fname (optional, otherwise loads from CONFIG_PATH env var)
+##'@example config$parameters_seir$gamma
+##'
+load_config <- function(fname) {
```

Comment:
Is the idea to have the same config for each report? a different project for each package? or something else?